guardrails/validators/bug_free_python.py

```diff
@@ -28,6 +31,26 @@ class BugFreePython(Validator):
     | Programmatic fix              | None                              |
     """
 
+    def __init__(self, on_fail: Optional[Callable] = None):
```

Comment:
@thekaranacharya if you're raising the warning during initialization and the format is common, then you can move this snippet to the parent class of each validator.

During the super().__init__ method the init method of the parent class will be executed and this warning can be raised there instead.

Reply:
Good point!